scripts/setup.sh

```
@@ -23,7 +34,129 @@ spec:
+        echo "Timed out waiting for LVMS CSV to exist"
+        exit 1
+    }
+    LVMS_CSV=$(oc get csv --no-headers -n openshift-storage | awk '/lvms/ { print $1 }')
+    wait_for_resource clusterserviceversion/${LVMS_CSV} jsonpath='{.status.phase}'=Succeeded 300 openshift-storage
+    wait_for_resource deployment/lvms-operator condition=Available 300 openshift-storage
+
+    # Apply LVMCluster configuration (requires operator CRDs to be installed)
+    oc apply -f prerequisites/lvms/lvms-config.yaml
+
+    # Wait for the storage class to be created and set it as default
+    retry_until 300 5 '[[ -n "$(oc get sc --ignore-not-found lvms-vg1)" ]]' || {
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)